apps/address-service/domains/address/schema/AddressSource.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Generated by `createschema address.AddressSource 'source:Text;'`
+ */
+
+const { Text, Relationship } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const access = require('@address-service/domains/address/access/AddressSource')
+const { SOURCE_ALREADY_EXISTS_ERROR } = require('@address-service/domains/address/constants')
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { AddressSource: AddressSourceApi } = require('@address-service/domains/address/utils/serverSchema')
+const isEmpty = require('lodash/isEmpty')
+
+const errors = {
+    SAME_SOURCE: {
+        mutation: 'createAddressSource',
+        code: BAD_USER_INPUT,
+        type: SOURCE_ALREADY_EXISTS_ERROR,
+        message: 'Source with the same address already exists',
+    },
+}
+
+const AddressSource = new GQLListSchema('AddressSource', {
+    schemaDoc: 'A model containing data on the particular building\'s address origin',
+    labelResolver: ({ source }) => source,
+    fields: {
+
+        source: {
+            schemaDoc: 'The string the address was found by (address origin)',
```

Comment:
It's not clear. is it a request query source?

Reply:
Yes, this is the string sent to the external provider to search address.

Reply:
It's a string used to find the address